data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/RecordDispatcher.java

```diff
@@ -184,8 +162,10 @@ private void onSubscriberSuccess(final KafkaConsumerRecord<String, CloudEvent> r
       .onComplete(finalProm);
   }
 
-  private void onSubscriberFailure(final KafkaConsumerRecord<String, CloudEvent> record, final Throwable exception,
+  private void onSubscriberFailure(final KafkaConsumerRecord<String, CloudEvent> record,
+                                   final Throwable exception,
                                    final Promise<Void> finalProm) {
+    logError("Failed to send event to subscriber", record, exception);
```

Comment:
No need for this log line, it's already in `composeSenderAndSinkHandler`

Reply:
Done!